game/static/game/css/auth.css

```
@@ -237,3 +237,42 @@ body {
+    box-shadow: 0 8px 24px rgba(0,0,0,0.5);
+    font-size: 0.95rem;
+    animation: toastEnter 0.4s cubic-bezier(0.175, 0.885, 0.32, 1.275) forwards;
+    display: flex;
+    align-items: center;
+    max-width: 350px;
+}
+.toast-success { border-left-color: #4caf50; }
+.toast-error { border-left-color: #ff6b6b; }
+.toast-warning { border-left-color: #f0c040; }
+
+@keyframes toastEnter {
```

Stylelint (17.11.0)

[error] Expected keyframe name "toastEnter" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)